contracts/src/v0.1/VRFCoordinator.sol

```diff
@@ -27,6 +27,7 @@ import "./CoordinatorBase.sol";
 /// generated by the requested oracle, and the requested number of
 /// random words are populated from it. Eventually, the random words
 /// are passed to consumer smart contract.
+
```

Comment:
Let's try to avoid changing audited code even though it does not change its functionality.

Reply:
Roger that. I pushed to trigger the actions. Next time, i'll use different approach